tutorials/printing.adoc

```diff
@@ -1,83 +1,213 @@
+
+Zahřejte tiskárnu na požadovanou teplotu a v rozhraní OctoPrint se přepněte na záložku xref:#_ruční-ovládání[ručního ovládání], a klikněte na tlačítko `Retract`.
+
+Po celou dobu vytahování mějte ruku poblíž struny, jelikož po jejím úplném vytažení má struna tendenci vystřelit a zamotat se. Vyjmutou strunu vložte do malého očka na špuli, aby se filament nezačal zamotávat. Buďte při manipulaci s filamentem opatrní. Špuli sundejte z držáku a filament odložte.
+
+
+==== Zavedení filamentu
+
+Pravidlo předehřáté tiskárny platí úplně stejně jako u vytahování filamentu.
+
+image::../images/filamentguide.svg[Jak má filament vypadat, 300, float="right"]
+
+Zkontrolujte zakončení filamentu na závady. Filament může být:
+
+. Nerovný či polámaný
+. Mít roztavené zakončení
+. Být "nahlodaný" od ozubení extruderu
+
+V každém případě takovou část uřízněte. 
+
+Zařízněte filament do špičky a vložte jej do otvoru na extruderu. Netlačte filament do tiskárny násilím, stačí jen vložit a jemně přitlačit a v OctoPrintu na záložce xref:#_ruční-ovládání[ručního ovládání] klikněte na tlačítko `Extrude`.
+
+Ve chvíli, kdy tiskárna uchopí filament můžete jej pustit. Extrudujte filament tak dlouho, dokud nezačne z trysky vytékat plast. Dále extrudujte plast tak dlouho, dokud nezačne z trysky vytékat plast správné barvy (v trysce mohly být pozůstatky filamentu jiné barvy).
+
+
+==== ABS Juice
+
+K vylepšení přilnavosti tiskové desky se používají různé materiály - lepidla, ABS Juice, laky. 
+V učebně je k dispozici ABS Juice, používejte, prosím, na tiskárny v učebně pouze ten. 
+Pokud tisknete předmět, který by se mohl v průběhu tisku odlepit, naaplikujte na desku jemnou vrstvu ABS Juice.
+
+
+=== Příprava tiskových dat
+
+Přípravou tiskových dat se zabývala kapitola link:#[Slicing]. TODO
+
+
+=== Tisk
+
+Nahrajte tisková data (ve formátu gcode) na Raspberry Pi použitím ovládacího prvku "Files" v levém dolním rohu rozhraní OctoPrintu.
+Pokud máte vše připraveno, můžete kliknout ikonku tisku u vašeho právě nahraného souboru.
+Pokud chcete zkontrolovat váš gcode, klikněte na ikonku složky.
+Tím se váš soubor nastaví jako aktivní, můžete si prohlédnout načtené statistiky, či ověřit jeho správnost na záložce "GCode Viewer" a tisk spustit až následným kliknutím na větší tlačítko "Print" v části rozhraní nazvané "State". 
+
+Pokud při tisku narazíte na jakýkoliv problém, konzultujte se cvičícím, nebo zkuste problém najít v xref:../troubleshooting/index#[troubleshootingu].
+
+
+=== Po dokončení tisku
+
+Pomocí nožíku se pokuste od tiskové desky odlepit celou základnu vašeho výtisku nebo alespoň její větší část.
+Velmi opatrně pak odlepte rukou výtisk z desky. 
+
+Proces odstraňování výtisku z tiskové desky neuspěchejte. 
+Dávejte pozor, abyste nepoškodili svůj výtisk nebo tiskovou desku.
+
+WARNING: Dodržujte pravidla bezpečné práce, abychom předcházeli zranění. Nože jsou ostré a hotendy tiskárny jsou po dokončení tisku stále horké a hrozí popálení. Buďte opatrní.
```

Comment:
Pokud ke zranění dojde, okamžitě kontaktuje cvičícího. Snažte se nezamazat tiskárnu moc od krve.

Reply:
A pokud budete moc krvácet, dejte pod sebe igelit, ať tu podlahu pak nemusíme vytírat. XD